src/Day1.hs

```diff
@@ -91,11 +93,14 @@ updatePlayerPos Input{keys=keys} (x,y) = (x',y')
 tileMap :: State -> Map (Int, Int) Tile
 tileMap st = case scene st of
   Scene'Start -> text "DAY 1" White1 (screenW `div` 2 - 2, screenH `div` 2)
-  Scene'Play -> mergeTiles clear $ fromList [ ( (x,y), Tile Nothing Nothing (Just wh1) ) ]
+  Scene'Play -> mergeTiles playerTile ( mergeTiles testDoors ( mergeTiles testWalls clear ) )
```

Comment:
The order is backwards. It should be `mergeTiles clear (` ....

Reply:
When it was the reverse the player would go under the room tiles. I'll test it again. Maybe I was doing something wrong.